src/Akka.Persistence.Sql.Linq2Db/Config/JournalTableColumnNames.cs

```diff
@@ -83,44 +84,5 @@ public override int GetHashCode()
             hashCode.Add(Manifest);
             return hashCode.ToHashCode();
         }
-        public static readonly string FallBack = @"tables.journal
```

Comment:
Baking hardwired HOCON config is a bad idea, this creates 4 copies of the same HOCON config in 4 different place